sktime/forecasting/lagllama.py

```diff
@@ -0,0 +1,348 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements an adapter for the LagLlama estimator for intergration into sktime."""
+
+__author__ = ["shlok191"]
+
+import subprocess
+
+from sktime.forecasting.base import _BaseGlobalForecaster
+
+
+class LagLlamaForecaster(_BaseGlobalForecaster):
+    """Base class that interfaces the LagLlama forecaster.
+
+    Parameters
+    ----------
+    weights_url : str, optional (default="time-series-foundation-models/Lag-Llama")'
```

Comment:
I would propose to use as parameter name `model_path` since it could also be a local model I assume.

Reply:
Made the change in new commits :)